### ISSUE TITLE
Layman overlay is missing manifest for cool-retro-term-9999-r1

### ISSUE THREAD

**Issue by AdShea** [Swordfish90/cool-retro-term#262]
I'm not sure who is in charge of the layman overlay (for Gentoo) but one of the ebuilds is missing in the manifest and portage complains.

Running 

    ebuild cool-retro-term-9999-r1.ebuild digest

will correct this.

**Comment by tgerczei** [comment]
I am. Thanks for pointing this out - I've actually long since added it and then shamefully forgot to commit. (BTW my contact data is in the repo metadata in layman)

**Comment by AdShea** [comment]
Thanks for doing the overlay, I must be blind missing your contact info.

**Comment by tgerczei** [comment]
@AdShea : You're most welcome - I'm glad it's useful :)

**Comment by shmerl** [comment]
Is the project developed in general still? The last change to the code was more than a half a year ago.
